server/src/services/asset.service.ts

```diff
@@ -286,6 +287,11 @@ export class AssetService {
         })),
       );
     } else {
+      if (!dto.trashReason) {
+        dto.trashReason = AssetTrashReason.DELETED;
```

Comment:
I'd prefer just destructuring `trashReason` in line 278 as well.

Reply:
Done